frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -43,13 +43,22 @@ function ReaderPaging:init()
     self.key_events = {}
     if Device:hasKeys() then
         self.key_events.GotoNextPage = {
-            {Input.group.PgFwd}, doc = "go to next page",
+            { {"RPgFwd", "LPgFwd", "Right" } }, doc = "go to next page",
```

Comment:
Was there an issue with simply adding Left/Right to the PgBack/PgFwd input groups?

Reply:
input.group.PgFwd/PgBack is also used in another places eg. scrolltextwidget, keyvaluepage etc.

Reply:
Okay, that's what I thought, thanks for the confirmation ;).